react/components/Tooltip/tooltip.stories.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import { withKnobs, text, boolean, select, number } from "@storybook/addon-knobs"
```

Comment:
you are not using the boolean
```suggestion
import { withKnobs, text, select, number } from '@storybook/addon-knobs'
```